recipes/nuraft/all/conandata.yml

```diff
@@ -0,0 +1,16 @@
+sources:
+  "cci.20220801":
+    url: "https://github.com/eBay/NuRaft/archive/50031e1b230569ccaa46008fc09d401e3e5c5985.tar.gz"
+    sha256: 225c30d04d9e067010b4a9b3340961ae89c30bee00e9762fcdc651401f890e41
+patches:
+  "cci.20220801":
+    - patch_file: "patches/0001-cmake_patches.patch"
+      patch_description: "Provide CMake correct dependency discovery logic."
+      patch_type: "conan"
+      base_path: "source_subfolder"
+      sha256: "2f847610184e680fa2d40e496c2094fc10aa55c4b652e56240c765cfc3a51a75"
+    - patch_file: "patches/0002-boost-asio.patch"
+      patch_description: "Use Boost::Asio rather than Asio solo project."
```

Comment:
We have an existing pattern when both are supported

https://github.com/conan-io/conan-center-index/blob/67ff53732892a8ab958fc6ffca79a374976efdcd/recipes/restinio/all/conanfile.py#L19

Reply:
@szmyd I think is a good idea supporting both, if possible, as Chris described. Your recipe is not the first with such case.

Reply:
Will take a look :+1: